### PR TITLE
fix lint issues in the apis/abac directory (except latest.go)

### PR DESCRIPTION
**Pull request by logicalhan**
**What this PR does / why we need it**:
Fixes the linting errors in pkg/apis/abac/ directory (with the exception of latest.go, which has a interesting todo)

**Release Note**
```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/abac/v0/register.go [hunk 1]
```diff
@@ -40,9 +41,12 @@ func init() {
 }
 
 var (
+	// SchemeBuilder is the scheme builder with scheme init functions to run for this API package
 	// TODO: move SchemeBuilder with zz_generated.deepcopy.go to k8s.io/api.
 	// localSchemeBuilder and AddToScheme will stay in k8s.io/kubernetes.
```
Comment (cheftako) [inline pkg/apis/abac/v0/register.go]: Can we move this line down to stay with localSchemeBuilder and AddToScheme?

File: pkg/apis/abac/v0/register.go [hunk 2]
```diff
@@ -22,9 +22,10 @@ import (
 	"k8s.io/kubernetes/pkg/apis/abac"
 )
 
+// GroupName is the group name use in this package
 const GroupName = "abac.authorization.kubernetes.io"
 
-// GroupVersion is the API group and version for abac v0
+// SchemeGroupVersion is the API group and version for abac v0
```
[the remark this answers is not in the document]
Comment (logicalhan) [reply pkg/apis/abac/v0/register.go]: sure thing

File: pkg/apis/abac/v0/register.go (hunk 1 above)
Comment (logicalhan) [reply pkg/apis/abac/v0/register.go]: done

**Comment by mcrute** [comment]
This looks ready to merge... adding @liggitt for approval

/unassign
/assign @liggitt
/lgtm

**Pull request merged**